libs/documentation/src/lib/pages/layout/data.service.ts

```diff
@@ -9,15 +9,15 @@ export class DataService {
     this.data = registrationData._embedded.results;
     if (this.data) {
       if (search.filter) {
-        if (search.filter.keyword) {
+        if (search.filter && search.filter[0].keyword) {

```

Comment:
The if statement on line 11 covers undefined possibility for search.filter, so the initial part here is not needed